### PR TITLE
Create Dockerfile and .dockerignore for frankly

### PR DESCRIPTION
**Pull request by nzwulfin**
This is a cool project, but I'm not a node programmer.  So instead of installing npm and bower locally I created a Dockerfile and containerized it.  All the commands are straight from the README.

The idea would be check out the repo, follow the README for updating the code to point to the right repos, then build the container and run it.  If you change the repos you're reporting on, you'll need to rebuild the container.

Let me know if this completely unsolicited PR is broken or shady!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
 <body>
   <frank-ly header="🚂🚃🚃💨"
       full-repo-names
-      repos='["notwaldorf/emoji-rain", "notwaldorf/emoji-selector", "polymer/polymer"]'
+      repos='["nzwulfin/vagrant-cluster", "projectatomic/atomic-site", "nzwulfin/contrib"]'
```
Comment (notwaldorf) [inline index.html]: Would you mind reverting this change? I'm selling emoji! ✨💝🐼:)

Comment (nzwulfin) [inline index.html]: Gah sorry, dirty branch!  I'll clean that up but probably not til Monday.

Comment (notwaldorf) [inline index.html]: No worries!

**Comment by nzwulfin** [comment]
The edit was buried in the commit not standalone that I could pull so I reverted that file to your master as a fix.  If it's still not clean, I can do more drastic git stuff.  It's only two files that should be added here.

**Comment by JakeChampion** [comment]
LGTM! :+1:

**Pull request merged**